model2vec/train/classifier.py

```diff
@@ -246,6 +267,15 @@ def evaluate(
 
         return report
 
+    def get_most_important_tokens(self, text: str) -> list[tuple[str, float]]:
```

Comment:
I think you can leave these functions out? I don't really see why they exist, except that they strongly couple the modules, where they could be decoupled before. i.e., if this function doesn't exist, there is no need to ever change this code if the explainability module changes, but now there suddenly is.